mentii/mentii
# Config problem bad steps

Provides 3 different ways of displaying bad steps [all bad steps in the tree, 3, 1]. Allows toggle of bad step color.

**Significant Changes**
1. Added button to toggle color for bad steps
2. Added dropdown to set number of bad steps to show before "Uh Oh" message is displayed

**How to Test**
1. Login and go to problems page
2. Open browser console to see step limit print out (will remove before merge)
3. Step through problems like normal/correcting them and change configurations

**Notes**
- Changing the step limit will change the number of bad steps to display for the **next** bad step tree that occurs. So for example if you set the limit to 3 in the beginning, encounter a bad step, then midway change the limit to 1, the current bad step tree will step till 3, but the next time the bad step tree will have the limit of 1.
